Modules/cmathmodule.c

```diff
@@ -957,12 +957,12 @@ cmath.log
 
 log(z[, base]) -> the logarithm of z to the given base.
 
-If the base not specified, returns the natural logarithm (base e) of z.
+If the base not specified, returns the natural logarithm of z.
```

Comment:
@OTheDev Good spot! It may be worth creating a separate PR to fix (i.e. a PR that just adds the missing "is"). That at least would be uncontroversial.

As things stand, I'm inclined to reject this PR - I don't think it represents an overall increase in clarity.